generators/app/templates/_package.json

```diff
@@ -38,7 +38,7 @@
     "semver": "5.5.0",
     "shelljs": "0.8.2",
     "through2": "2.0.3",
-    "generator-jhipster": ">=5.4.0",
+    "generator-jhipster": "6.0.0-beta.0",
```

Comment:
need to take care about this version

Reply:
I created #35 so we don't forget to change that once JHipster 6 has been released.